src/tools/lifter/models/l3mod/m0_mod_stepLib.sml

```diff
@@ -0,0 +1,129 @@
+structure m0_mod_stepLib =
```

Comment:
Libs should have signatures, either inline (see Pablo's relSynth file for syntax) or in another file. I'm not sure about what happens when you `open` this library without a signature. Do you inherit all the `open`s or not? That's why I'm using
```sml
local
  open stuff
  val private_stuff = "mine"
in (* local *)
  public_stuff
end (* local *)
```
to be sure that `open`s don't pollute the global namespace. Maybe that's useless, I don't know (yet) :innocent:

Reply:
no, you're absolutely right. these kind of things do pollute. i also checked it already.